CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-09-18
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240918.
+
+### Announcements
```

Comment:
Did k8s 1.31 make this release? @kaarthis

Reply:
No we could not make it . It has to be next release.